apps/polls/templates/meinberlin_polls/poll_404.html

```diff
@@ -4,14 +4,17 @@
 {% block error_status %}404{% endblock %}
 
 {% block error_description %}
-    {% trans 'Participation is not possible at the moment.' %}
+    <p>
```

Comment:
unfortunately this results in invalid html, as https://github.com/liqd/a4-meinberlin/blob/dd5b43723b55d5a7c2f3b6e8398ef4968858b3d0/meinberlin/templates/base_errors.html already encapsulates the error_description in a `<p>` and paragraphs are not allowed nested in html